acme/account.go

```diff
@@ -43,15 +45,63 @@ func KeyToID(jwk *jose.JSONWebKey) (string, error) {
 	return base64.RawURLEncoding.EncodeToString(kid), nil
 }
 
+// PolicyNames contains ACME account level policy names
+type PolicyNames struct {
+	DNSNames []string `json:"dns"`
+	IPRanges []string `json:"ips"`
+}
```

Comment:
Should we include CommonName checks for ACME?

Reply:
I thought a bit about that, but I don't think it's necessary to include them here. 

When an ACME order is finalized we perform a check if all names in the CSR are the same as in the ACME Order. The Common Name is included in that check. This also means that the challenge for the Common Name was successfully completed by the client, so the Common Name is then either a routable IP or reachable DNS.